lib/src/test/resources/http4s/server/form-params-017.txt

```diff
@@ -98,12 +98,12 @@ trait ModelRoutes {
             val responseOpt = for {
               id <- req.getFirst("id")
               model <- req.getFirst("model").flatMap(f => _root_.io.circe.parser.decode[io.apibuilder.http4s.test.models.Model](f).toOption)
-              enum <- req.getFirst("enum").flatMap(f => _root_.io.circe.parser.decode[io.apibuilder.http4s.test.models.Enum](f).toOption)
+              `enum` <- req.getFirst("`enum`").flatMap(f => _root_.io.circe.parser.decode[io.apibuilder.http4s.test.models.Enum](f).toOption)
```

Comment:
```
req.getFirst("`enum`")
```

I'm not sure why there are backticks around enum inside the double quotation marks.

Reply:
anytime we generate a reserved word, we place it in backticks per scala spec.  This looks right to me given the addition of `enum` as a keyword

Reply:
@mbryzek the string in `req.getFirst("`enum`")` shouldn't be placed in backticks though

Reply:
ah - that's right

Reply:
I implemented a fix. I changed this:
```
val originalName = ScalaUtil.quoteNameIfKeyword(field.originalName)
```

to

```
val originalName = field.originalName
```